packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -276,21 +152,20 @@ contract FederatedAttestations is
    * @dev Adds attestation info to the arrays in order of provided trustedIssuers
    * @dev Expectation that only one attestation exists per (identifier, issuer, account)
    */
-  function lookupAttestations(
-    bytes32 identifier,
-    address[] calldata trustedIssuers,
-    bool includeRevoked
-  ) external view returns (uint256[] memory, address[] memory, uint256[] memory, address[] memory) {
+  function lookupAttestations(bytes32 identifier, address[] calldata trustedIssuers)
+    external
+    view
+    returns (uint256[] memory, address[] memory, uint256[] memory, address[] memory)
```

Comment:
I figured there will be lots of changes to make in the lookup functions, which there is a separate issue up for, so I only modified them enough to compile successfully for now

Reply:
I think param reordering may fit well with this PR since this is where the attestation struct order + events are being changed?  Fine to leave it as is for now but please add a TODO in the locations where ordering should be changed so we don't forget to do this later 🙏

Reply:
added TODOs :)